Composer/packages/client/src/recoilModel/atoms/botState.ts

```diff
@@ -47,6 +47,22 @@ const emptyDialog: DialogInfo = {
   skills: [],
   isFormDialog: false,
 };
+type lgStateParams = { projectId: string; lgFileId: string };
+
+export const lgFileState = atomFamily<LgFile, lgStateParams>({
+  key: getFullyQualifiedKey('lg'),
+  default: () => {
+    return {} as LgFile;
```

Comment:
why does it return an empty object? If someone try to access a non-existing lg file this should return undefined not an empty object.